packages/babel-compiler/babel-compiler.js

```diff
@@ -300,6 +300,16 @@ BCp._inferHelper = function (
   merge(babelOptions, babelrc, "presets");
   merge(babelOptions, babelrc, "plugins");
 
+  const babelEnv = (process.env.BABEL_ENV ||
+                    process.env_NODE_ENV ||
```

Comment:
I believe so. From the babel docs:

`The env key will be taken from process.env.BABEL_ENV, when this is not available then it uses process.env.NODE_ENV if even that is not available then it defaults to "development".`

https://babeljs.io/docs/usage/babelrc/#env-option